### PR TITLE
Various edits to contributing guidelines

### PR DESCRIPTION
**Pull request by DNin01**
### Changes

Makes a few changes to the contributing guidelines meant to improve the first-time contributor experience.

Some notable changes:
- Changed the headings
- Referenced octocat/Spoon-Knife for first-time contributors
- Improved step-by-step walkthrough

The diff looks pretty intense, but that's just from things being moved around a bit.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
 Another way to suggest an idea or to ask a question is to start a [discussion](https://github.com/ScratchAddons/ScratchAddons/discussions).
 
-## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+## Making contributions
+
+Contributions are welcome! If you know how to fix an issue, you can create a pull request to propose the necessary changes which we can then add to Scratch Addons.
+
+### Learn more about making addons
+
+If you're making a new addon, our ["Creating an Addon" FAQ page](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
```
Comment (Hans5958) [inline .github/CONTRIBUTING.md:21]: Instead of "FAQ page", let's just do "documentation page" or "page".

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
+5. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on) from the `master` branch.
+6. [Commit and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes to your new branch.
+7. If you haven't already, go to your fork on the website and enable workflows in Actions on your fork. This will enable an automation that formats (cleans up) your code if necessary.
+8. Finally, go to the website to [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). There should be a notification giving you a shortcut to open a pull request. (If you don't see it, you can manually open a pull request by clicking "New pull request", then "Compare across forks" and selecting your fork and branch next to "head repository". Click "create pull request" to go to the next step.) Finally, fill out the form and click "create pull request" to finish! We will review your changes and talk about any further improvements if necessary.
```
Comment (Hans5958) [inline .github/CONTRIBUTING.md:38]: Super picky review (safe to ignore): Separate this to multiple numbers, or a sublist?

**Review by Hans5958 — CHANGES_REQUESTED** [review]
The first one is the one that is important (plain wrong). Anything else LGTM.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (DNin01) [reply .github/CONTRIBUTING.md:21]: Oops!

Comment (Hans5958) [reply .github/CONTRIBUTING.md]: I would say a guide would be a little too much, but it is fine for me.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
+## Reporting bugs and suggesting features
+
+Found a bug? You can open an [issue](https://github.com/ScratchAddons/ScratchAddons/issues) to report it! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
 Also, feel free to ask questions, suggest new features, and more.
 
 Another way to suggest an idea or to ask a question is to start a [discussion](https://github.com/ScratchAddons/ScratchAddons/discussions).
 
-## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+## Making contributions
```
Comment (Hans5958) [inline .github/CONTRIBUTING.md:15]: I would suggest write this as "code contributions", or something with "code" in it. Contributions can come in any way, such as "reporting bugs and suggesting features" (the previous section).

**Review by Hans5958 — CHANGES_REQUESTED** [review]
A little bit more. Also, can we link to the FAQ section?

https://scratchaddons.com/docs/faq/#contributing

**Review by Hans5958 — APPROVED** [review]
Looks good enough to me.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -1,28 +1,39 @@
...
+5. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on) from the `master` branch.
+6. [Commit and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes to your new branch.
+7. If you haven't already, go to your fork on the website and enable workflows in Actions on your fork. This will enable an automation that formats (cleans up) your code if necessary.
+8. Go to the website to [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). There should be a notification giving you a shortcut to open a pull request. (If you don't see it, you can manually open a pull request by clicking "New pull request", then "Compare across forks" and selecting your fork and branch next to "head repository". Click "create pull request" to go to the next step.)
```
Comment (WorldLanguages) [inline .github/CONTRIBUTING.md:38]: Some words inconsistently start with lowercase I think. Shouldn't it be `Create pull request`?

**Comment by WorldLanguages** [comment]
Looks good

**Review by WorldLanguages — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -1,28 +1,39 @@
...
-If you're making a new addon, our [FAQ page](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
+If you are familiar with JavaScript, HTML, and CSS, you can contribute by fixing bugs, making adjustments, or adding features. Just follow the steps below to open a pull request to propose your changes which we can then add to Scratch Addons.
+
+### Learn more about making addons
+
+If you're making a new addon, our ["Creating an Addon" guide](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
+
+### Opening a pull request
+
+We recommend using [Git](https://git-scm.com/) to commit changes. If you prefer GUIs, you can use [GitHub Desktop](https://desktop.github.com/) instead. You can also use the GitHub website if that's easier, but the other options speed up some of the process.
```
Comment (DNin01) [inline .github/CONTRIBUTING.md:25]: Should we have the GitHub website listed as an option?

Comment (WorldLanguages) [reply .github/CONTRIBUTING.md:25]: I think it's ok as is

**Comment by WorldLanguages** [comment]
Maybe we could get a third approval... Possibly @Samq64?

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,28 +1,39 @@
...
+Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CODE_OF_CONDUCT.md).
 
-Also, feel free to ask questions, suggest new features, and more.
+## Reporting bugs and suggesting features
 
-Another way to suggest an idea or to ask a question is to start a [discussion](https://github.com/ScratchAddons/ScratchAddons/discussions).
+Found a bug? Have an idea? You can open an [issue](https://github.com/ScratchAddons/ScratchAddons/issues) to report bugs and send feedback! But first, be sure to [search for existing issues](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
-## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+You can also suggest ideas or ask questions in [Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions).
```
Comment (Samq64) [inline .github/CONTRIBUTING.md:13]:
```suggestion
You can also open a [Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions) to suggest ideas or ask questions.
```

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (Samq64) [reply .github/CONTRIBUTING.md:25]:
```suggestion
We recommend using [Git](https://git-scm.com/) to commit changes. If you prefer GUIs, you can use [GitHub Desktop](https://desktop.github.com/) instead. If you don't want to install anything, you can use the GitHub website, but we recommend one of the other options if possible.
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -1,28 +1,39 @@
...
-
-(These instructions are for using [Git](https://git-scm.com/) to commit your changes, but you can also use the website, [GitHub Desktop](https://desktop.github.com/), [Visual Studio Code](https://code.visualstudio.com/), or any other software that lets you make your changes.)
+3. If you already have a fork, make sure to [sync its `master` branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to ensure your fork of Scratch Addons is up to date with the source code.
+4. [Clone your fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) and [load it into your browser](https://github.com/ScratchAddons/ScratchAddons#from-source) so you can make and test your changes. If you've already cloned your fork, remember to [pull down any changes](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository#fetching-changes-from-a-remote-repository) to make sure you're up to date.
+5. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on) from the `master` branch.
```
Comment (Samq64) [inline .github/CONTRIBUTING.md:35]:
```suggestion
5. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on) from the `master` branch. This makes things easier later on if you have multiple pull Request open.
```

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -1,28 +1,39 @@
...
+6. [Commit and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes to your new branch.
+7. If you haven't already, go to your fork on the website and enable workflows in Actions on your fork. This will enable an automation that formats (cleans up) your code if necessary.
+8. Go to the website to [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). There should be a notification giving you a shortcut to open a pull request. (If you don't see it, you can manually open a pull request by clicking "New pull request", then "Compare across forks" and selecting your fork and branch next to "head repository". Click "Create pull request" to go to the next step.)
```
Comment (Samq64) [inline .github/CONTRIBUTING.md:38]:
```suggestion
8. Go to the website to [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). There should be a notification giving you a shortcut to open a pull request. If you don't see it, you can manually open a pull request by clicking "New pull request", then "Compare across forks" and selecting your fork and branch next to "head repository". Click "Create pull request"
```

Comment (DNin01) [reply .github/CONTRIBUTING.md]: I guess the parentheses could be removed, but "to go to the next step" sounds somewhat helpful to say.

**Review by Samq64 — APPROVED** [review]

**Review by apple502j — APPROVED** [review]

**Pull request merged**